src/bitmessagekivy/tests/telenium_process.py

```diff
@@ -0,0 +1,85 @@
+"""
+    Base class for telenium test cases which run kivy app as background process
+"""
+
+import os
+import shutil
+import tempfile
+from time import time, sleep
+
+from telenium.tests import TeleniumTestCase
+
+_files = (
+    'keys.dat', 'debug.log', 'messages.dat', 'knownnodes.dat',
+    '.api_started', 'unittest.lock'
+)
+
+tmp_db_file = (
+    'keys.dat', 'messages.dat'
+)
+
+
+def cleanup(home=None, files=_files):
+    """Cleanup application files"""
+    if not home:
+        home = tempfile.gettempdir()
+    for pfile in files:
+        try:
+            os.remove(os.path.join(home, pfile))
+        except OSError:
+            pass
+
+
+def set_temp_data():
+    """Set temp data in tmp directory"""
+    for file_name in tmp_db_file:
+        old_source_file = os.path.join(
+            os.path.abspath(os.path.dirname(__file__)), 'sampleData', file_name)
+        new_destination_file = os.path.join(os.environ['BITMESSAGE_HOME'], file_name)
+        shutil.copyfile(old_source_file, new_destination_file)
+
+
+class TeleniumTestProcess(TeleniumTestCase):
```

Comment:
I would make the two function from above, `cleanup` and `set_temp_data`, methods of this function, to avoid passing around directory.